### ISSUE TITLE
Extension for Firefox: strict_min_version (compatibility with Waterfox Classic)

### ISSUE THREAD

**Issue by grahamperrin** [sourcegraph/sourcegraph#7367]
The extension's preference page: 

`moz-extension://d0784587-8b24-41b0-ae73-f4a9fbbfba3c/options.html`

– **appears blank** in home-built [Waterfox Classic](https://www.waterfox.net/releases/) 2019.12 (20191210201058) on FreeBSD-CURRENT. Page source: 

```
<!DOCTYPE html>
<html>
    <head>
        <meta http-equiv="Content-type" content="text/html; charset=utf-8" />
        <title>Sourcegraph extension</title>
        <link href="./css/options-style.bundle.css" rel="stylesheet" type="text/css" />
    </head>
    <body>
        <script type="text/javascript" src="./js/options.bundle.js"></script>
    </body>
</html>
```

tl;dr historically, Waterfox 56.0 was based on Firefox 56.0.2. 

If you can't easily make the extension compatible, then please consider adding a `strict_min_version` to its manifest. 

Thanks
